app/model/DTO/Travel/Command.php

```diff
@@ -238,8 +239,9 @@ public function getTransportTypes() : array
     public function getTransportTypePairs() : array
     {
         $types = [];
+
         foreach ($this->transportTypes as $type) {
-            $types[$type->getShortcut()] = $type->getLabel();
+            $types[$type->toString()] = $type->getLabel();
```

Comment:
není tady bezpečnější udělat metodu `getKey()` protoze si dokazu predstavit, ze to string bude vracet i jen label

Reply:
Je to konzistentní s ostatními enumy, které v Hospodaření máme.